README.md

```diff
@@ -117,6 +118,36 @@ building your own cogs!
 
 Join us on our [Official Discord Server](https://discord.gg/red)!
 
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```

Comment:
This links to a 404 page <https://github.com/Cog-Creators/Red-DiscordBot/blob/V3/develop/CONTRIBUTING.md> which is not the correct location of our `CONTRIBUTING.md` file.